[Bug]: Vue 2.7 usage - runtime errors + `legacy` seems to be removed but not mentioned in migration guide

### Would you like to work on a fix?

- [ ] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

### Current and expected behavior

Although it's mentioned in migration guide that the package supports Vue 2.7 I have difficulties with making it working.
First - importing from `vue-chartjs/legacy` doesn't work any more:

```js
// Module not found: Error: Package path ./legacy is not exported from package ...
import { Bar as BarChart } from 'vue-chartjs/legacy'; 
```

Second - importing from 'vue-chartjs' throws missing property errors, even after changing them from `chart-data` to `data` etc.

![image](https://user-images.githubusercontent.com/15625235/205892892-5caafe48-2eae-43bd-a6f3-56663909aa0c.png)


### Reproduction

https://stackblitz.com/edit/github-tqxopy?file=src%2FApp.vue

### chart.js version

4.0.1

### vue-chartjs version

5.0.0

### Possible solution

_No response_

--- a/.size-limit.json
+++ b/.size-limit.json
@@ -1,7 +1,7 @@
 [
   {
     "path": "dist/index.js",
-    "limit": "1.9 KB",
+    "limit": "1.95 KB",
     "webpack": false,
     "running": false
   },

--- a/src/typedCharts.ts
+++ b/src/typedCharts.ts
@@ -14,6 +14,7 @@ import {
 import type { TypedChartComponent, ChartComponentRef } from './types.js'
 import { CommonProps } from './props.js'
 import { Chart } from './chart.js'
+import { compatProps } from './utils.js'
 
 export function createTypedChart<
   TType extends ChartType = ChartType,
@@ -36,11 +37,18 @@ export function createTypedChart<
       expose({ chart: ref })
 
       return () => {
-        return h(Chart, {
-          ref: reforwardRef as any,
-          type,
-          ...props
-        })
+        return h(
+          Chart,
+          compatProps(
+            {
+              ref: reforwardRef as any
+            },
+            {
+              type,
+              ...props
+            }
+          )
+        )
       }
     }
   }) as any

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -1,4 +1,4 @@
-import { isProxy, toRaw } from 'vue'
+import { isProxy, toRaw, version } from 'vue'
 import type {
   Chart,
   ChartType,
@@ -7,6 +7,13 @@ import type {
   ChartOptions,
   DefaultDataPoint
 } from 'chart.js'
+
+export const compatProps =
+  version[0] === '2'
+    ? <I extends {}, T extends {}>(internals: I, props: T) =>
+        Object.assign(internals, { attrs: props }) as unknown as I & T
+    : <I extends {}, T extends {}>(internals: I, props: T) =>
+        Object.assign(internals, props)
 
 export function toRawIfProxy<T>(obj: T) {
   return isProxy(obj) ? toRaw(obj) : obj

--- a/website/src/migration-guides/index.md
+++ b/website/src/migration-guides/index.md
@@ -21,6 +21,7 @@ If you are experiencing this problem with Jest, you should follow [this doc](htt
 - `chartOptions` props were renamed to `options`
 - unknown props will fall through to the canvas element.
 - `generateChart` were refactored and renamed to `createTypedChart`
+- Vue.js < 2.7 is no longer supported. If you want to use vue-chartjs with Vue < 2.7 you have to lock your version to 4.x.
 
 ## Migration from v3 to v4
 

